soh/src/code/z_play.c

```diff
@@ -1441,6 +1443,18 @@ void Gameplay_Draw(GlobalContext* globalCtx) {
     CLOSE_DISPS(gfxCtx);
 }
 
+time_t day_seconds() {
```

Comment:
```suggestion
time_t Gameplay_GetRealTime() {
```
This follows our naming conventions better.

Reply:
Done c: